Framework/include/Baseline.h

```diff
@@ -328,6 +340,14 @@ class Baseline
         bool pass_qcdCR_1b_1t = pass_qcdCR && NGoodBJets_pt30 >= 1  && ntops >= 1;
         bool pass_qcdCR_2b    = pass_qcdCR && NGoodBJets_pt30 >= 2;
 
+        // Now combine all relevant baselines into a single bool
+        // Here we select the main baselines for all three channels
+        // If all four booleans are false, then the event is considered a "lost cause"
+        lostCauseEvent &= !passBaseline0l_Good &&
```

Comment:
This is what I'm nervous about from my comments in Analyzer. It would be nice if we don't have to hard code the selections used in this bool.